.php_cs

```diff
@@ -107,10 +153,15 @@ $config->setRules(
         'single_line_comment_style' => true,
         'single_quote' => true,
         'space_after_semicolon' => true,
+        'standardize_increment' => true,
         'standardize_not_equals' => true,
+        'strict_comparison' => true,
+        'strict_param' => true,
```

Comment:
👎 to `strict_comparison` and `strict_param`

While I support doing this, it's is very dangerous and should be used at the developer's discretion.

Reply:
This is "sane defaults" - not for everything. This is intended to be a starting point, and can always be changed.

Reply:
If this was for a linter, I would agree. But php-cs-fixer typically is used in a pre-commit hook and is applied automatically, it's meant to be invisible. I'd rather avoid having all the head-scratching that comes when `if ($float == 0)` stopped working after simply doing a commit.

Reply:
Let's have it but mention in the README that this can change how the code works.